### ISSUE TITLE
Ajouter avatar GH

### ISSUE THREAD

**Issue by adriens** [adriens/domaine-nc-javafx#13]

**Comment by adriens** [comment]
:information_source: Sauf erreur de ma part , je ne vois pas vos contributions en code :thinking:

**Comment by Davnox900** [comment]
je suis actuellement entrain de faire un thread pour l'affichage des domaines, j'avais fais une version basic de l'interface de l'application.

**Comment by adriens** [comment]
> je suis actuellement entrain de faire un thread pour l'affichage des domaines, j'avais fais une version basic de l'interface de l'application.

yes, très bonne approche de décorréler UI et back. Pensez au reactive.

Egalement hésitez pas à faire appel à Chat GPT pour coder plus vite ⚡

**Comment by Davnox900** [comment]
d'ailleurs que voulez vous dire par avatar gh?

**Comment by adriens** [comment]
> d'ailleurs que voulez vous dire par avatar gh?

Yep, indice pour toi : 
![image](https://user-images.githubusercontent.com/5235127/233209650-9c5bc2ba-1214-404b-95bc-eef7e6b6b97b.png)

**Comment by adriens** [comment]
Joli @Davnox900 ... fan de [Red Hat](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) ?

![image](https://user-images.githubusercontent.com/5235127/234215125-eafdcaf6-8739-49ce-85f9-4e9efa3355d3.png)
